src/main/java/net/montoyo/mcef/client/ClientProxy.java

```diff
@@ -167,9 +167,11 @@ public void onInit() {
                 libs.add(System.getProperty("sun.arch.data.model").equals("64") ? "d3dcompiler_47.dll" : "d3dcompiler_43.dll");
                 libs.add("libGLESv2.dll");
                 libs.add("libEGL.dll");
+                libs.add("chrome_elf.dll");
```

Comment:
chrome_elf.dll/.so is a new file added since Chrome 55 and up (we're on 67 now)

without this we get a "library failed to load dependencies error"